src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform #{$trans-duration}, color #{$trans-duration};
+
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Don't remember create variables for color
```suggestion
      color: $c-someColor;
```